pkg/controller/datadogagentdeployment/clusteragent.go

```diff
@@ -27,6 +27,8 @@ import (
 	"github.com/DataDog/datadog-operator/pkg/controller/utils/comparison"
 )
 
+const availabilityRequeueAfterDuration = 5 * time.Second
```

Comment:
maybe we can consider to reuse the default requeue duration: https://github.com/DataDog/datadog-operator/blob/24eb8b44b92fa948a6db5a4ffa5105430fa2181b/pkg/controller/datadogagentdeployment/datadogagentdeployment_controller.go#L41

Reply:
indeed, thanks